mk/flags.mk

```diff
@@ -10,16 +10,16 @@ LDFLAGS ?=
 
 CROSS_COMPILE ?=
 
-CC      := $(CROSS_COMPILE)gcc
-CPP     := $(CC) -E
-CXX     := $(CROSS_COMPILE)g++
-AR      := $(CROSS_COMPILE)ar
-AS      := $(CROSS_COMPILE)as
-LD      := $(CROSS_COMPILE)ld
-NM      := $(CROSS_COMPILE)nm
-OBJDUMP := $(CROSS_COMPILE)objdump
-OBJCOPY := $(CROSS_COMPILE)objcopy
-SIZE    := $(CROSS_COMPILE)size
+CC      ?= $(CROSS_COMPILE)gcc
+CPP     ?= $(CC) -E
+CXX     ?= $(CROSS_COMPILE)g++
+AR      ?= $(CROSS_COMPILE)ar
+AS      ?= $(CROSS_COMPILE)as
+LD      ?= $(CROSS_COMPILE)ld
+NM      ?= $(CROSS_COMPILE)nm
+OBJDUMP ?= $(CROSS_COMPILE)objdump
+OBJCOPY ?= $(CROSS_COMPILE)objcopy
+SIZE    ?= $(CROSS_COMPILE)size
```

Comment:
Yep, see Travis [logs](https://travis-ci.org/embox/embox/jobs/74988530#L92):

    $ export CC=gcc

That's how Travis [works](http://docs.travis-ci.com/user/languages/c/#Choosing-compilers-to-test-against) for C projects.

> Testing against two compilers will create (at least) 2 rows in your build matrix. For each row, Travis CI C builder will export the `CC` env variable to point to either `gcc` or `clang`.